deepl/deepl.py

```diff
@@ -143,6 +143,8 @@ async def __translate(self, script: str) -> str:
             ).split("-")[0]
 
             res = str((await output_textbox.all_inner_texts())[0])
+            # the extra \n is generated by <p> tag because every line is covered by it
+            res = res.replace("\n\n", "\n")
```

Comment:
I'd like to go through every p tag without using all_inner_texts (which returns new lines, except for other functons) function but I don't know how to do that properly. However, this solution seems to work.